mirrord-layer/src/lib.rs

```diff
@@ -97,7 +98,10 @@ fn init() {
         HOOK_SENDER = Some(sender);
     };
 
-    let enabled_file_ops = ENABLED_FILE_OPS.get_or_init(|| config.enabled_file_ops);
+    let enabled_file_ops =
+        ENABLED_FILE_OPS.get_or_init(|| (config.enabled_file_ops || config.enabled_file_ro_ops));
+    let _ = ENABLED_FILE_RO_OPS
```

Comment:
If you don't set enabled_file_ops to true then the hooks won't be enabled in `enable_hooks` fn

Reply:
but I added ||?

Reply:
I mean the local variable of `enabled_file_ops` for line 106, right now its `let _ = ENABLED_FILE_RO_OPS`

Reply:
the get or init sets it